core/src/main/java/tech/kwik/flupke/server/HttpServerRequest.java

```diff
@@ -32,6 +32,8 @@ public interface HttpServerRequest {
 
     HttpHeaders headers();
 
+    String host();
```

Comment:
I think I would prefer to call this "authority()".
https://www.rfc-editor.org/rfc/rfc3986#section-3.2 states that authority   = [ userinfo "@" ] host [ ":" port ] (but for HTTP/3 userinfo should not be used), so the authority contains the host and port (if not default). Therefore, it feels wrong to call this method "host", as it does not only return the host.
I know that in HTTP/2 and HTTP/1 the header was called "Host" and did contain host+port, but that doesn't mean we have to copy that misnomer ;-)